### PR TITLE
Stronger cors filter

### PR DESCRIPTION
**Pull request by amontenegro**

### CODE REVIEW

**Comment by amontenegro** [comment]
@rcpeters all set, please verify.

File: orcid-core/src/main/java/org/orcid/core/web/filters/CrossDomainWebManger.java [hunk 1]
```diff
@@ -32,41 +32,43 @@
 public class CrossDomainWebManger {
 
     private static final String LOCALHOST = "localhost";
-    
-    Pattern p = Pattern.compile(".*/public/.*|.*/public_widgets/.*|.*/userStatus\\.json|.*/lang\\.json");
-    
-    @Value("${org.orcid.security.cors.allowed_domains:orcid.org}")
+
+    Pattern p = Pattern.compile("/public/.*|/public_widgets/.*|.*/userStatus\\.json|.*/lang\\.json");
```
Comment (rcpeters) [inline orcid-core/src/main/java/org/orcid/core/web/filters/CrossDomainWebManger.java]: public should be ^/public and what about userStatus and lang.json?

**Comment by amontenegro** [comment]
Hey @rcpeters please check this out.

Thanks

**Comment by rcpeters** [comment]
:+1:

**Pull request merged**